CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 * Add multi-lang component support scaffolding.
 * Fix type errors in TypeScript checking awsx-classic properties.
 * Ensure that FargateService and EC2Service default `continueBeforeSteadyState` to false.
+* Allow tagging public subnets on VPC for security group matching
```

Comment:
Should we add equivalents to this for all the other subsets of resources we're creating as well?  Or at least also `privateSubnetTags`?